.claude/skills/ecc-harness-patterns/SKILL.md

```
@@ -0,0 +1,102 @@
+        "cursor": {"context_key": "editor", "supports": ["streaming", "streaming_chunks"]},
+        "codex": {"context_key": "project", "supports": ["completion"]},
+    }
+    
+    def normalize_request(self, harness_id: str, request: dict) -> dict:
+        """Convert harness-native request to local-llm-server format"""
+```
+
+### 2. Session Lifecycle Hooks
+ECC's `/hooks/session-*` pattern applied to local-llm-server:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)